akka-docs-dev/rst/scala/migration-guide-1.0-2.x-scala.rst

```diff
@@ -552,6 +556,12 @@ Example
       // This no longer works!
       val someFileSink = SynchronousFileSink(new File("."))
 
+      // This no longer works!
+      val yetAnotherFileSource = Source.file(new File("."))
```

Comment:
I think we can have it here, with a comment that it was like this in milestone versions. Might help users to migrate which are using milestone versions.